src/cljs/rems/actions/vote.cljs

```diff
@@ -92,37 +93,61 @@
                                                      :attachments attachments
                                                      :on-finished on-finished}])}]))
 
+;; TODO too difficult, consider changing data model
+;; TODO consider moving to a util
+(defn- get-reviewers [application]
+  (for [event (:application/events application)
+        :let [type (:event/type event)]
+        :when (= type :application.event/review-requested)
+        reviewer (:application/reviewers event)]
+    reviewer))
+
+(defn- get-potential-voters [application]
+  (case (-> application :application/workflow :workflow/voting :type)
+    :handlers-vote
+    (get-in application [:application/workflow :workflow.dynamic/handlers])
+
+    :reviewers-vote
+    (get-reviewers application)
+
+    nil))
+
 (defn votes-summary
   [application]
   (let [votes (get-in application [:application/votes])
         summary (frequencies (vals votes))
-        voters (get-in application [:application/workflow :workflow.dynamic/handlers])
+        voters (get-potential-voters application)
         voter-by-userid (build-index {:keys [:userid]} voters)
         voters-by-vote (build-index {:keys [val] :value-fn (comp voter-by-userid key) :collect-fn conj} votes)]
-    [:div.my-3
-     [:h3 (text :t.applications/votes)]
-
-     [:div.container-fluid
-      (for [[vote n] (sort-by val summary)
-            :let [n-pct (* 100 (/ n (count voters)))
-                  vote-voters (get voters-by-vote vote)]]
-        ^{:key vote}
-        [:div.form-group.row
-         [:label.col-sm-3.col-form-label (text (keyword (str "t" ".applications.voting.votes") vote))]
-         [:div.col-sm-9.form-control (goog.string/format "%.2f%% (%s)"
-                                                         n-pct
-                                                         (->> vote-voters
-                                                              (mapv application-util/get-member-name)
-                                                              (str/join ", ")))]])
-
-      (let [n (- (count voters) (count votes))
-            n-pct (* 100 (/ n (count voters)))
-            missing-voters (->> voters
-                                (remove (comp (set (keys votes)) :userid))
-                                (mapv application-util/get-member-name)
-                                (str/join ", "))]
-        [:div.form-group.row
-         [:label.col-sm-3.col-form-label (text :t.applications.voting.votes/empty)]
-         [:div.col-sm-9.form-control (goog.string/format "%.2f%% (%s)"
-                                                         n-pct
-                                                         missing-voters)]])]]))
+    (when (seq voters)
+      [:div.my-3
+       [:h3 (text :t.applications/votes)]
+
+       [:div.container-fluid
+        (doall (for [[vote n] (sort-by val summary)
```

Comment:
```clojure
(into [:<>] 
      (for [[vote n] (sort-by val summary) ...
```
is a little simpler to write and you can omit react keys like `^{:key vote}` inside the list comprehension. for a small data structure, any performance hits should be negligible when it needs to re-render

Reply:
It's only simpler if React key is omitted? But yes here we can leave it out as there's only a few rows ever.